packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantBadges/RestaurantBadge.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <span
+        :class="{
+            [$style['c-restaurantBadge']]: true,
+            [$style['c-restaurantBadge--isLarge']]: isLarge
+        }"
+        :style="`color:${textColour}; background-color:${backgroundColour};`"
```

Comment:
I can see why you're doing it this way (as it does make the component super flexible), but would there not be a finite number of colour combinations we should adhere to as part of our design system?

If there is, then I would definitely go with using modifiers, as the only problem with making the component this flexible is that it means that colour combinations that haven't been approved by the design team can be used by consumers.

Reply:
Yeah I do agree. I'm happy to go with that approach. @1itvinka you may want to do similar for the icon text you're working on.

Reply:
👍